invenio_rdm_records/services/schemas/record.py

```diff
@@ -36,6 +44,20 @@ def validate_scheme(scheme):
         raise ValidationError(_("Invalid persistent identifier scheme."))
 
 
+class InternalNoteSchema(Schema):
+    """Schema for internal notes."""
+
+    id = SanitizedUnicode(load_default=lambda: RecordIdProviderV2.generate_id())
```

Comment:
I have doubts about using the RecordIDProvider in the schema of the record, maybe someone has an idea for something more elegant
I need to identify the notes within the record, otherwise there is no way of keeping track

Reply:
See my comment above